### DISCUSSION TITLE
[BUG] test.md is missing

### DISCUSSION THREAD

**Discussion by gjtorikian** [excid3/noticed#489] [Q&A] [answered]
The README says:

```
Delivery methods we officially support:

...

[Test](https://github.com/excid3/noticed/blob/main/docs/delivery_methods/test.md)
```

But https://github.com/excid3/noticed/blob/main/docs/delivery_methods/test.md is completely blank.

**Comment by excid3** [comment, marked as answer]
Just updated, but it's only meant for testing and probably not something you'll use.
